test/examples/aws_handler.js

```diff
@@ -0,0 +1,26 @@
+// begin lambda connection
```

Comment:
Not specific to this file.  

Our other documentation examples live in https://github.com/mongodb/node-mongodb-native/tree/main/test/integration/node-specific/examples.  Could we bring the old tests into the new top-level folder?  This does require working with the docs team though, since they point to https://github.com/mongodb/node-mongodb-native/tree/main/test/integration/node-specific/examples in our repo to pull the examples from.

Reply:
Waiting on docs to respond to what impact this move would have.

Reply:
I'm open to filing a follow up ticket.  Alternatively, we could move the tests from `test/examples` into the existing folder, and then file a follow up ticket to move all of them to`test/examples` later?  Both ways work for me.

Reply:
@durran (tagging you for visibility)

Reply:
I've moved to the other examples dir now.

Reply:
https://jira.mongodb.org/browse/NODE-4579